webpack/global_test_setup.js

```diff
@@ -8,7 +8,7 @@ global.console.error = (error, stack) => {
 };
 
 // Increase jest timeout as some tests using multiple http mocks can time out on CI systems.
-jest.setTimeout(300000);
+jest.setTimeout(process.env.JEST_TIMEOUT || 10000);
```

Comment:
```suggestion
jest.setTimeout(process.env.JEST_TIMEOUT || 20000);
```

Reply:
2nd test run passed on github with 10k..20k should be good.. 👍🏽